DependencyInjection/Configuration.php

```diff
@@ -68,6 +68,17 @@ public function getConfigTreeBuilder()
                             ->end()
                         ->end()
                     ->end()
+                    ->children()
+                        ->arrayNode('symmetric')
+                            ->useAttributeAsKey('name')
+                            ->prototype('array')
+                                ->children()
+                                    ->scalarNode('path')->isRequired()->end()
```

Comment:
Since this option contains the *path* to the key file may be it would be nice to name it 'key_path'?